routers/api/v2/tokens.go

```diff
@@ -64,3 +66,33 @@ func (r *apiV2Router) CreateServiceToken(ctx *gin.Context) {
 		Token: token,
 	})
 }
+
+// DELETE: /api/v2/tokens/service/:id
+// Response:
+// Headers:  Authorization <- token
+func (r *apiV2Router) InvalidateServiceToken(ctx *gin.Context) {
+	tokenID := ctx.Param("id")
+	if len(tokenID) == 0 {
+		r.logger.Debug("token id must be provided in request")
+		models.SendAPIError(ctx, http.StatusBadRequest, "token id must be provided")
+		return
+	}
+
+	err := r.authorizer.InvalidateServiceToken(ctx, tokenID)
+	if err != nil {
+		switch errors.Cause(err) {
+		case services.ErrInvalidID:
+			r.logger.Error("service token id is not valid", zap.Error(err))
```

Comment:
```suggestion
			r.logger.Debug("service token id is not valid", zap.Error(err))
```

This is user error, `Error` logging level should not be used here.